app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -33,6 +33,11 @@ public UserService(Mapper dozerMapper,
         this.passwordEncoder = passwordEncoder;
     }
 
+    /**
+     * User를 등록한다. 만약 이메일이 중복되면 예외를 발생시킨다.
+     * @param registrationData User의 데이터.
+     * @return 등록된 User를 반환한다.
+     */
```

Comment:
이메일이 중복되는 경우를 예외로 기술해주는 것은 어떨까요?

```suggestion
    /**
     * 사용자를 등록하고, 등록한 사용자를 반환합니다.
     * @param registrationData 사용자 데이터
     * @return 등록된 사용자
     * @throws DuplicatedException 이메일이 중복되면 예외를 던집니다. 
     */
```